auto_nag/scripts/missing_beta_status.py

```diff
@@ -30,47 +30,36 @@ def ignore_date(self):
     def get_autofix_change(self):
         return self.autofix_status
 
+    @staticmethod
+    def _is_affected(status):
+        return status and status not in ("unaffected", "?")
+
     def handle_bug(self, bug, data):
         bugid = str(bug["id"])
-        central = bug[self.status_central]
+        nightly = bug[self.status_nightly]
         release = bug[self.status_release]
         doc = self.get_documentation()
 
-        if central == release:
-            if release != "verified":
-                self.autofix_status[bugid] = {
-                    "comment": {
-                        "body": "Change the status for beta to have the same as nightly and release.\n{}".format(
-                            doc
-                        )
-                    },
-                    self.status_beta: central,
-                }
-            else:
-                # if the two status are different, we don't know what to set
-                # if this verified on nightly and release, we cannot say
-                # per say if this is verified
-                self.autofix_status[bugid] = {
-                    "comment": {
-                        "body": "Since it has been verified for nightly and release, is it verified for beta too?\n{}".format(
-                            doc
-                        )
-                    },
-                    self.status_beta: "?",
-                }
+        if self._is_affected(nightly) and self._is_affected(release):
+            self.autofix_status[bugid] = {
+                "comment": {
+                    "body": f"Since nightly and release are affected, beta will likely be affected too.\n${doc}"
+                },
+                self.status_beta: "affected",
+            }
         else:
             self.autofix_status[bugid] = {
                 "comment": {
-                    "body": "Since the status are different for nightly and release, what's the status for beta?\n{}".format(
-                        doc
-                    )
+                    "body": f"Since the status is marked as `${nightly}` for nightly and release, is it `${nightly}` for beta too?\n${doc}"
```

Comment:
The status here could be different between nightly and release, shouldn't we mention both potential values?

Reply:
We do that at L55:

https://github.com/mozilla/relman-auto-nag/blob/0cc3879a25b6c6e6235a78d9c892b5d04f3f0bce/auto_nag/scripts/missing_beta_status.py#L53-L55

Reply:
Oops, I didn't notice the second part!
Nit: maybe rewrite as:
```Python
if nightly == release:
    body = XXX
else:
    body = XXX
```

Reply:
This is empirical evidence that it needs to be clearer ;)

Hopfully e9f884eee0e433a49ad5e67d6ad6a152d96eb2c0 will make it clearer. I avoided adding a top level if-statement to make it clear it is just a message customization.